tests/backend/unit/test_project_structure.py

```
@@ -0,0 +1,702 @@
+4. __init__.py files are present where needed
+
+The tests verify both the backend (FastAPI in app/) and frontend (React in frontend/)
+structures, as well as test directories and documentation.
+"""
+
+import ast
+import importlib
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List, Set, Tuple
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)